db/v-2/tables/lnk-vulnerability-user.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="lnk-vulnerability-user-1" author="frolov" context="dev or prod">
+        <createTable tableName="lnk_vulnerability_user">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="vulnerability_id" type="bigint">
+                <constraints foreignKeyName="fk_vulnerability_user_vulnerability" references="vulnerability(id)" nullable="false" deleteCascade="true"/>
+            </column>
+            <column name="user_id" type="bigint">
+                <constraints foreignKeyName="fk_lnk_vulnerability_user_user" references="user(id)" nullable="false" deleteCascade="true"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet id="lnk-vulnerability-user-2" author="frolov">
+        <addUniqueConstraint tableName="lnk_vulnerability_user" columnNames="user_id, vulnerability_id"/>
```

Comment:
can it be moved to `changeSet id="lnk-vulnerability-user-1"`?

Reply:
Yes. changed it